src/themes/saas/components/input.ts

```diff
@@ -29,6 +29,7 @@ export const Input = defineMultiStyleConfig({
     auth: (props: any) => ({
       field: {
         fontWeight: '500',
+        py: '25px',
```

Comment:
I am not sure if is a good practice to add the padding for all the auth inputs types. I mean, maybe there are a form where the input type doesn't need this 25px padding? 

If you consider that this is needed for all the input auth, ignore this comment and resolve conversation :)

Reply:
We will do a PR fixing base styles for some components